packages/expo-updates/android/src/main/java/expo/modules/updates/manifest/ManifestResponse.java

```diff
@@ -15,6 +15,6 @@ public ManifestResponse(Response response) {
   }
 
   public String header(String name) {
-    return mResponse.header(name);
+    return mResponse.header(name, null);
```

Comment:
let's just additionally implement `public String header(String name, String defaultValue)` to keep the behavior of this wrapper as true to `okhttp3.Response` as we can